cmd/kubelet/app/server.go

```diff
@@ -261,6 +260,9 @@ HTTP server: The kubelet can also listen for HTTP and respond to a simple API
 			// add the kubelet config controller to kubeletDeps
 			kubeletDeps.KubeletConfigController = kubeletConfigController
 
+			if err := checkPermissions(); err != nil {
+				klog.Error(err)
```

Comment:
As kubernetes/enhancements#1602, could you change the log here to a structured log?